sonyci/cli.py

```diff
@@ -84,6 +87,35 @@ def search(
     print(dumps(result))
 
 
+@app.command()
+def download(
+    ctx: Context,
+    id: Annotated[str, Argument(..., help='The SonyCi ID of the file to download')],
+    proxy: Annotated[ProxyType, Option('--proxy', '-p', help='Download ')] = None,
+    output: Annotated[
+        Path, Option('--output', '-o', help='The path to download the file to')
+    ] = None,
+):
+    """Download a file from Sony CI"""
+    ci = SonyCi(t=ctx.parent.params['token'])
+    log.trace(f'download id: {id} proxy: {proxy} output: {output}')
+    result = ci.asset_download(id)
+    link = result['location']
+    if proxy:
+        for p in result['proxies']:
+            if p['type'] == proxy.value:
+                log.debug(f'found proxy {proxy.value}')
+                link = p['location']
+                break
+        assert link != result['location'], f'proxy {proxy} not found'
```

Comment:
should we opt for `raise` over assert? I think `assert` stops working in production if `__debug__` is false.

Reply:
I didn't know that about `__debug__`! Will `raise` it instead!